pkg/cmd/cli/secrets/new.go

```diff
@@ -18,7 +19,23 @@ import (
 	"github.com/spf13/cobra"
 )
 
-const NewSecretRecommendedCommandName = "new"
+const (
+	NewSecretRecommendedCommandName = "new"
+
+	newLong = `Create a new secret based on a key file or on files within a directory.
+  `
+
+	newExamples = `  // Create a new secret named my-secret with a key named foo
+  $ %[1]s my-secret /path/to/foo
+
+  // Create a new secret named my-secret with a key named bar, overwriting the name of the key on disk
+  $ %[1]s my-secret bar=/path/to/foo
```

Comment:
I worry that particular example will make it less clear what this is doing, but I don't feel strongly.  Whatever it takes to get merged.